README.tfdoc.hcl

```diff
@@ -593,21 +593,28 @@ section {
     title   = "Module Outputs"
     content = <<-END
       The following attributes are exported in the outputs of the module:
+    END
 
-      - **`module_enabled`**
-
+    output "module_enabled" {
+      type        = bool
+      description = <<-END
         Whether this module is enabled.
+      END
+    }
 
-      - **`bucket`**
-
+    output "bucket" {
+      type        = object(bucket)
+      description = <<-END
         All attributes of the created `google_storage_bucket` resource.
+      END
+    }
 
-      - **`iam`**
-
+    output "iam" {
+      type        = list(iam)
```

Comment:
```suggestion
      type        = object(iam)
```

Reply:
@thiesen I think this might be a list. Here's the output from `output.tf`:
```
# remap iam to reduce one level of access (iam[]. instead of iam[].iam.)
output "iam" {
  description = "The iam resource objects that define access to the GCS bucket."
  value       = { for key, iam in module.iam : key => iam.iam }
}
```

Reply:
yes this is correct!

Reply:
sorry for the noise! I still need to wrap my head around this kind of stuff - for me it was definitely a map/object :sweat_smile: